terraform/schema.yaml

```diff
@@ -936,16 +956,60 @@ variables:
                     - "Use Public Subnet"
     type: boolean
     default: true
-    title: "Provision Bastion Instance"
+    title: "Provision Resource Manager Private Endpoint"
     description: "Provision a bastion compute instance on a public subnet to provide access to the WebLogic Server compute instances on a private subnet. If not selected, you must check the status of the compute instances after the stack job is complete."
```

Comment:
Why is this mentioning creating a bastion compute instance?

Reply:
This needs to be changed. I have noted this down to talk with Vanaja for better wording.